src/main/java/org/junit/rules/RuleChain.java

```diff
@@ -77,8 +77,12 @@ private RuleChain(List<TestRule> rules) {
      *
      * @param enclosedRule the rule to enclose.
```

Comment:
Please change this line to `@param enclosedRule the rule to enclose; must not be {@code null}.`